posGlobalOffsetBiasX: 0.0
posGlobalOffsetBiasY: 0.0
posGlobalOffsetBiasZ: 0.0
noisePosXVar: 0.0
noisePosYVar: 0.0
noisePosZVar: 0.0
noiseVelXVar: 0.5
noiseVelYVar: 0.5
noiseVelZVar: 0.1
noiseOffXVar: 0.02
noiseOffYVar: 0.02
noiseOffZVar: 0.02
velocityDecayTime: 3.0
offsetDecayTime: 30.0
falsePositiveThresholdSigma: 2.0
initialUncertaintyPosXY: 100.0
initialUncertaintyPosZ: 10.0
initialUncertaintyVelXY: 1.0
initialUncertaintyVelZ: 0.5
initialUncertaintyOffsetXY: 1.0
initialUncertaintyOffsetZ: 3.0
